keyboards/handwired/swiftrax/bumblebee/bumblebee.h

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2021 Swiftrax <swiftrax@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#define KC_NO XXX
+
+#include "quantum.h"
+
+ #define LAYOUT_default( \
```

Comment:
If this macro supports every matrix location on the PCB, it should be named `LAYOUT_all`.

```suggestion
 #define LAYOUT_all( \
```